test/jdk/com/sun/jdi/lib/jdb/Debuggee.java

```diff
@@ -212,4 +204,17 @@ public void close() throws IOException {
         }
     }
 
+    /**
+     * Parses debuggee output to get listening transport and address, printed by `launch=echo`.
+     * Returns null if the string specified does not contain required info.
+     */
+    private static JDWP.ListenAddress parseLaunchEchoListenAddress(String debuggeeOutput) {
+        Pattern listenRegexp = Pattern.compile("Listen Args: \\b(.+)\\b \\b(.+)\\b");
```

Comment:
"Listen Args:" should be a static final and also referenced above where the "echo" command is used.